sdk/program/src/rent.rs

```diff
@@ -3,40 +3,54 @@
 //! [rent]: https://docs.solana.com/implemented-proposals/rent
 
 #![allow(clippy::integer_arithmetic)]
-//! configuration for network rent
 
 use {
     crate::{clock::DEFAULT_SLOTS_PER_EPOCH, clone_zeroed, copy_field},
     std::mem::MaybeUninit,
 };
 
+/// Configuration of network rent.
 #[repr(C)]
 #[derive(Serialize, Deserialize, PartialEq, Copy, Debug, AbiExample)]
 pub struct Rent {
-    /// Rental rate
+    /// Rental rate in lamports/byte-year.
     pub lamports_per_byte_year: u64,
 
-    /// exemption threshold, in years
+    /// Amount of time (in years) a balance must include rent for the account to
+    /// be rent exempt.
     pub exemption_threshold: f64,
 
-    // What portion of collected rent are to be destroyed, percentage-wise
+    /// The percentage of collected rent that is burned.
+    ///
+    /// Valid values are in the range [0, 100]. The remaining percentage is
+    /// distributed to validators.
     pub burn_percent: u8,
 }
 
-/// default rental rate in lamports/byte-year, based on:
-///  10^9 lamports per SOL
-///  $1 per SOL
-///  $0.01 per megabyte day
-///  $3.65 per megabyte year
+/// Default rental rate in lamports/byte-year.
+///
+/// This calculation is based on:
+/// - 10^9 lamports per SOL
+/// - $1 per SOL
+/// - $0.01 per megabyte day
+/// - $3.65 per megabyte year
 pub const DEFAULT_LAMPORTS_PER_BYTE_YEAR: u64 = 1_000_000_000 / 100 * 365 / (1024 * 1024);
 
-/// default amount of time (in years) the balance has to include rent for
+/// Default amount of time (in years) the balance has to include rent for the
+/// account to be rent exempt.
 pub const DEFAULT_EXEMPTION_THRESHOLD: f64 = 2.0;
 
-/// default percentage of rent to burn (Valid values are 0 to 100)
+/// Default percentage of collected rent that is burned.
+///
+/// Valid values are in the range [0, 100]. The remaining percentage is
+/// distributed to validators.
 pub const DEFAULT_BURN_PERCENT: u8 = 50;
 
-/// account storage overhead for calculation of base rent
+/// Account storage overhead for calculation of base rent.
+///
+/// This is an approximate constant number of bytes required to store an account
+/// of any size. It is added to an accounts data length when calculating
```

Comment:
Actually I believe this is the exact number of bytes required for an account with no data.

Reply:
I've changed it. Can't remember what I was thinking with "approximate".